src/aio/IoUring.zig

```diff
@@ -40,6 +40,7 @@ pub fn isSupported(op_types: []const type) bool {
         op.* = switch (Operation.tagFromPayloadType(op_type)) {
             .nop => std.os.linux.IORING_OP.NOP,
             .fsync => std.os.linux.IORING_OP.FSYNC, // 5.4
+            .poll, .child_exit => std.os.linux.IORING_OP.POLL_ADD, // 5.13 (child_exit uses waitid if available)
```

Comment:
Does IORING_OP_POLL_REMOVE have to be supported, does it overlap with cancel?

Reply:
I have corrected the required versions
POLL_ADD requires 5.13 only when used with POLL_ADD_MULTI otherwise 5.1 is sufficient
POLL_REMOVE requires 5.1 and we can use ASYNC_CANCEL instead